assets/js/modules/adsense/components/widgets/ConnectAdSenseCTATileWidget.js

```diff
@@ -52,11 +51,9 @@ export default function ConnectAdSenseCTATileWidget( { Widget, WidgetNull } ) {
 		keyMetrics?.includes( KM_ANALYTICS_ADSENSE_TOP_EARNING_CONTENT )
 	) {
 		return (
```

Comment:
Ideally I think this component would be like the one for GA4, and then the top earning content tile widget would use 2 instances of `whenActive` (one for adsense and one for GA4) as we do here https://github.com/google/site-kit-wp/blob/67114ba29e75238d0ee3ae14e7683dbef16fd9c2/assets/js/modules/adsense/components/dashboard/DashboardTopEarningPagesWidget.js#L222-L225

This is perhaps more relevant for #6248, although maybe some of the prep could be done here? Arguably, this component shouldn't be rendered at all (it's currently registered into the area) since there is no tile to show for AdSense yet, so this would make more sense to be reworked into the fallback for that tile (which could maybe be scaffolded here) similar to the top countries tile or similar?

If you feel it's out of scope, we can do it in a separate issue too.

Reply:
SGTM, I've gone ahead and made these changes for the Connect AdSense CTA widget as well and included a skeleton Top Earning Content widget as well. I've also updated the QAB to reflect this.